server/utils/licenceStatus.js

```diff
@@ -130,10 +130,16 @@ function getCaStageState(licence) {
 
 function getEligibilityStageState(licence) {
     const {excluded, exclusion} = getExclusionTaskState(licence);
-    const {insufficientTime, crdTime} = getCrdTimeState(licence);
+    const {insufficientTime, crdTime, insufficientTimeContinue} = getCrdTimeState(licence);
     const {unsuitable, suitability, exceptionalCircumstances} = getSuitabilityState(licence);
     const eligibility = getEligibilityState(unsuitable, excluded, [exclusion, crdTime, suitability]);
-    const eligible = isEligible({excluded, insufficientTime, unsuitable, exceptionalCircumstances});
+    const eligible = isEligible({
+        excluded,
+        unsuitable,
+        exceptionalCircumstances,
+        insufficientTime,
+        insufficientTimeContinue
```

Comment:
insufficientTime shouldn't be part of eligible, because we use 'not eligible' to signal 'stop, no further processing required' (probably need to clean up some names in this aspect later) but for insufficient time it will proceed, either to RO or DM